query/common/src/test/groovy/org/codice/ddf/admin/common/fields/base/BaseObjectFieldTest.groovy

```diff
@@ -118,4 +118,25 @@ class BaseObjectFieldTest extends Specification {
         then:
         subFieldOfInnerField.getValue() == 'valueChange'
     }
+
```

Comment:
May be a little tedious, but should add a check for duplicity of errors in all these tests to make sure we never return the same error twice on accident?